test/test_since_resolution.sh

```
@@ -0,0 +1,287 @@
+  printf "    echo '[]'\n"                       >> "$mock_dir/gh"
+  printf '    ;;\n'                              >> "$mock_dir/gh"
+  printf '  *"pulls/comments/42/replies"*)\n'   >> "$mock_dir/gh"
+  printf "    echo '[]'\n"                       >> "$mock_dir/gh"
+  printf '    ;;\n'                              >> "$mock_dir/gh"
+  printf '  *) exit 1 ;;\n'                     >> "$mock_dir/gh"
+  printf 'esac\n'                               >> "$mock_dir/gh"
+  chmod +x "$mock_dir/gh"
+}
+
+run_script() {
+  PATH="$mock_dir:$ORIGINAL_PATH" bash "$script" "$@"
```

Shellcheck (0.11.0)

[warning] script is referenced but not assigned.

(SC2154)